bottomline_recordandreplay/README.md

```diff
@@ -0,0 +1,53 @@
+# Bottomline Mainframe Record and Replay
+
+## Overview
+
+Bottomline's Mainframe Record and Replay solution is non-invasive in its capability to monitor 3270/5250 users via network traffic.  Bottomline helps customers monitor users and systems in ways more advanced than traditional approaches with our industry leading Fraud and Financial Crime solutions.
+
+This integration is for existing Bottomline and Datadog customers, creates a logger in Bottomline's Record and Replay using a pre-built JSON message format, which can then be sent to Datadog using the standard agent. 
+
+### Monitoring Capabilities
+
+1. Mainframe Users: Record and Replay the users session and log information about what the user did in the session.
+2. Mainframe: User Response Time
+3. Mainframe: Resource Response Time 
```

Comment:
I would suggest changing the description to a full sentence for consistency and clarity. For example, "Mainframe: Record the user's response time."